xml/deployment_prep_aarch64_raspi.xml

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+
+<sect1 version="5.0" xml:id="sec.aarch64.raspi"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>&rpi;</title>
+
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+
+ <para>
+  &slsreg; is the first enterprise Linux distribution to fully support the
+  inexpensive &rpi; single-board computer. &sls; 15.1 supports only the &rpi; 3
+  Model B and B+.
+ </para>
+
+ <para>
+  There are two different ways to deploy &sls; onto &rpi; hardware:
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    Copy a pre-installed system image onto a MicroSD card.
+   </para>
+   <para>
+    This is the most common method for deploying operating systems on the
+    &rpi; platform.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Boot from USB media and install onto another drive.
+   </para>
+   <para>
+    This is the more conventional way to install &sls;, although it is less
+    common for &rpi; operating systems.
+   </para>
+  </listitem>
+ </itemizedlist>
+ 
+ <para>
+  There are several points to note regarding both methods.
+ </para>
+ 
+ <sect2>
+  <title>Deploying from a Preinstalled Image</title>
+  <para>
+   This method is the simplest and easiest method. However, it allows little
+   customisation of the product being deployed except for values such as network
+   and account settings.
+  </para>
+  <para>
+   &suse; provides a pre-configured bootable image of &sle; for &rpi; 3 series
+   hardware. This comes with the lightweight IceWM desktop and the Btrfs file
+   system, with compression enabled to improve performance and reduce wear on
+   MicroSD media.
+  </para>
+  <para>
+   A MicroSD card with a minimum size of 8 GB is recommended. Faster cards will
+   give better system performance. On the first boot, the operating system
+   automatically expands the filesystem to fill the card. This means that the
+   first boot will be substantially slower than subsequent boots.
+  </para>
+  <para>
+   &sle; 15.1 adds support for the &rpi; 3 Model B+, which was not fully
+   functional using &sle; 15. Apart from this, the installation process is the
```

Comment:
Are you sure about the B+? What is not fully functional in 15 GA?

Reply:
I tried SLE 15 on a 3 B+ and there was no networking connectivity. Neither wifi nor Ethernet could get an address and manually setting an address with YAST did not work.